conjur/utils.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+"""
+Utils module
+
+This module holds all util functions
+"""
+import sys
+
+# pylint: disable=too-few-public-methods
+class Utils:
+    """
+    Utils
+
+    This class holds all common utils needed across the CLI/Client
+    """
+    @classmethod
+    def insecure_message(cls):
+        """
+        Method for printing the an insecure warning prompt
+        when user uses --insecure flag
+        """
+        # pylint: disable=line-too-long
+        sys.stdout.write("*" * 100 + "\n")
+        sys.stdout.write("Warning: Running the command with '--insecure' makes your system vulnerable to security attacks!\n")
+        sys.stdout.write("*" * 100 + "\n")
```

Comment:
use `print` with multiline string:
```suggestion
        print(f"""{"*" * 100}
                 Warning: Running the command with '--insecure' makes your system vulnerable to security attacks!
                 {"*" * 100}""")
```

Reply:
removed